core/crypto/key_type.hpp

```diff
@@ -22,19 +22,19 @@ namespace kagome::crypto {
     UNSUPPORTED_KEY_TYPE_ID,
   };
 
-  enum class SupportedKeyTypes : uint32_t {
-    BABE = 'babe',
-    GRAN = 'gran',
-    ACCO = 'acco',
-    IMON = 'imon',
-    AUDI = 'audi'
-  };
-
   /**
    * @brief Key type identifier
    */
   using KeyTypeId = uint32_t;
 
+  namespace supported_key_types {
+    static constexpr KeyTypeId kBabe = 1650549349u;
```

Comment:
Plsease, add comment about meaning of values, i.e. 
```... = 1650549349u; // "babe"```

Other way is over `reinterpret_cast`, i.e.
```static constexpr KeyTypeId kBabe = *reinterpret_cast<uint32_t*>("babe"); // NOLINT```